### PR TITLE
Fixes, final updates and testing for chapters 7 - 13 (14 later)

### PR DESCRIPTION
**Pull request by ghost**
Creating this PR to correct any issues that occur during final testing. Some issues of note are
- Fixing latex preamble errors such as
```
\DeclareMathOperator{\sech}{sech}
\DeclareMathOperator{\csch}{csch}
```
Which should converted to
```
\newcommand{\sech}{\operatorname{sech}}
\newcommand{\csch}{\operatorname{csch}}
```
There are also other errors such as ```\vv, \divv and \curl``` . I noticed that they are similar to the above commands but modifying them in the same way doesn't work. These are the originals I found in a header file. ```\curl```might need another package but I am not sure.
```
\DeclareMathOperator{\curl}{curl}
\DeclareMathOperator{\divv}{div}
```
- I will convert tikz images to pg-code in webwork on a different branch as there was a lot more work here than I expected (will be starting on ch 1-6 anyways so it won't happen here anytime soon).

TODOs
- [ ] Fix preample issues.
- [x] Add ```usepackage("amsmath")``` to all asymptote images that do not already have it.
- [x] Asymptote images will be tested in a full compile and verified in the html if all files compile correctly.
- [x] sec_triple_int should be able to be fully incorporated without issues now since there were too many changes originally for git to merge it properly.
- [x] Wrap a \<sidebyside\> around images in a webwork problem (will do the same for ch 1-6 in PTX repo).
- [x] Clean up a lot of old style caption xrefs.
- [x] font code in asymptote images will be removed.
- [x] add missing image code.
- [x] Clean up a lot of old style caption xrefs.

### CODE REVIEW

**Comment by sean-fitzpatrick** [comment]
Those commands for hyperbolic trig functions (sech, csch) could probably be left as is now that we're loading the amsmath package in all asymptote images.

The others you can deal with at indicated. These are just commands that are new for chapter 14, so not previously added to index.ptx.

Let me know if you're not sure about one and I can tell you how it should be defined.

**Comment by ghost** [comment]
Right those were chapter 14 things. I did do some testing with them in chapter 14 but we can deal with them there for now.

**Comment by ghost** [comment]
- Did 2 extra hours over then what I put into the time sheet today will add those onto tomorrows hours.

**Comment by ghost** [comment]
It appears using 
```
\newcommand{\sech}{\operatorname{sech}}
\newcommand{\csch}{\operatorname{csch}}
```
Breaks a lot of latex commands so I am not sure how to get around it.

**Comment by ghost** [comment]
I'm not really sure why but any new commands we have tried to add to the preamble we add causing asymptote to crash in mbx. The vv command you gave me:```\newcommand{vv}``` was one that did it this time. I also did not get a log file or any intermediate file either this time. I don't really know how asymptote relates to latex works but I am very confused as to why latex commands are causing a separate language entirely to crash in so many ways.

**Comment by sean-fitzpatrick** [comment]
Asymptote relies on LaTeX for the formatting of math in labels.
Any macros used need to be loaded into Asymptote so it knows how to process them. The mbx script should be loading everything from index.ptx into the top of each asy file before processing.
Could be something wrong with the command?

**Comment by ghost** [comment]
That would be my guess. The modified hyperbolic ones have not worked for me either. Good to know that latex does work with asymptote that helps clear up some confusion I had. Still not sure why those declaremath commands fail as they do work correctly in the pretext math tags.

**Comment by ghost** [comment]
Just pushed a PR that fixed some broken xrefs. There were a very small number from the earlier chapters which I did fix here but I am not sure if they are relevant anymore in the other repo.

**Comment by ghost** [comment]
Finished testing everything in chapter 7-13 except the webwork problems as I was having an issue getting them to compile on ubuntu. Will try some things and test to make sure they look correct. I did correct some small things as I went but all the images are looking good and the text is also looking good. Will also check for missing images once I get webwork working again. Still lost on the latex command issues. Added 2-3 ToDo's about some old xrefs I couldn't figure out as well.

**Comment by ghost** [comment]
Having inconsistencies with extracting webwork. I was getting an error that a webwork problem must have a statement tag but every one of them did and it wouldn't compile them. It appears to be a server issue since that error went away when I switched to the aimath server. I wonder if it has something to do with the recent work on the image integration stuff. 

The other thing I noticed is I will run the set of commands in a clean workspace and sometimes I get all the webwork problems compiled in the static form (which I want) and other times it is blank which is the same result as not extracting the webwork but just compiling the ptx. I will continue to investigate.

**Comment by ghost** [comment]
It seems webwork is compiling fine now but I do have to use the other server still. Everything does appear to look correct and function properly in static mode (minus images of course) and this completes my main testing of all these chapters. I did not appear to find any missing images either so all that remains is the latex preample issue. I left a comment in the other PR on this. But once that is done we can be finished with his repo and focus on the PTX repo which I will start doing here. I may have left a lot of comments here but I don't think I left too many things for you to actually do.

- Fixed a small issue with old todo comments such as
 \"can't use \&#xf4; in a WW. It leads to ô, which is not in the base64 charset\"
cause a massive number of repeated, useless spam errors saying it doesn't like the symbol. I have simply deleted these lines.

- Fixed lines that used \underlinespace in webwork to use \<fillin\/> instead. Removed quote tags around 1 set of \<fillin\/> lines as they don't seem to work in them.

- Left ToDos for files that used 3d images in webwork (both tikz with addplot3 and asymptote with graph3) that they will not be supported for now. Commenting out huge chunks of text doesn't work unless you use a ton of quotes, otherwise I would have done that. A lot of these webwork problems don't randomize anything so unless the value of being able to submit a solution through webwork is very valuable, these could be converted to plain exercises as they are in chapter 14.

**Comment by sean-fitzpatrick** [comment]
I see one thing unchecked on your checklist: what preamble issues are there to take care of?

File: src_pp/sec_polarcalc.ptx [hunk 1]
```diff
@@ -1911,7 +1909,6 @@
             <pg-code>
             </pg-code>
             </setup>
-          <!-- can't use &#xe7; in a WW. It leads to ç, which is not in the base64 charset -->
```
Comment (sean-fitzpatrick) [inline src_pp/sec_polarcalc.ptx:1914]: These comments should stay. We addressed L'Hospital by changing the spelling. Here we still need the special character.

File: src_pp/sec_sequences.ptx [hunk 2]
```diff
@@ -815,7 +815,7 @@
             <figure xml:id="fig_seq5">
               <caption>A plot of a sequence in <xref ref="ex_seq5">Example</xref>, part 2.</caption>
           <!-- START figures/fig_seq5.tex -->
-              <image xml:id="img_seq5" width="47%">
```
Comment (sean-fitzpatrick) [inline src_pp/sec_sequences.ptx:818]: Why is width removed here? It doesn't seem to be in a sidebyside.

File: src_pp/sec_series.ptx [hunk 3]
```diff
@@ -498,7 +498,7 @@
               <figure xml:id="fig_series2">
                 <caption>Scatter plots relating to the series in <xref ref="ex_series2">Example</xref>.</caption>
             <!-- START figures/fig_series2a.tex -->
-                <image xml:id="img_series2" width="47%">
```
Comment (sean-fitzpatrick) [inline src_pp/sec_series.ptx:501]: Also for several images here. Shouldn't the width attribute be kept?

File: src_pp/sec_taylor_poly.ptx [hunk 4]
```diff
@@ -133,7 +133,7 @@
   <figure xml:id="fig_taypolyintrob">
     <caption>Plotting <m>f</m>, <m>p_2</m> and <m>p_4</m>.</caption>
     <!-- START figures/fig_taypolyintrob.tex -->
-    <image xml:id="img_taypolyintrob" width="47%">
+    <image xml:id="img_taypolyintrob">
```
Comment (sean-fitzpatrick) [inline src_pp/sec_taylor_poly.ptx]: More widths being dropped..

File: src_pp/sec_triple_int.ptx [hunk 5]
```diff
@@ -1286,1678 +1286,1680 @@
...
+              -->
+              <!-- START figures/fig3d_proj_3D.asy -->
               <image xml:id="img_trip2bb1_3D">
-              <description></description>
-            <asymptote>
-
-              usepackage("amsmath")
-              
-              import graph3;
+                <description></description>
```
Comment (sean-fitzpatrick) [inline src_pp/sec_triple_int.ptx:1300]: Somewhere around here the diff on GitHub goes weird again.
After this block line numbers are off by a couple hundred.

File: src_pp/sec_triple_int.ptx [hunk 6]
```diff
@@ -1286,1678 +1286,1680 @@
...
+              <image xml:id="img_trip3a2_3D">
+                <description></description>
+                <asymptote>
 
-              usepackage("amsmath")
-              
-              import graph3;
+                  import graph3;
 
-              //ASY file for fig3d_projb3D.asy in Chapter 13
```
Comment (sean-fitzpatrick) [inline src_pp/sec_triple_int.ptx:1415]: At this point it's trying to do a diff for two different images!

**Comment by ghost** [comment]
That would be the latex commands not working issue. I have so far been unable to get any of them functioning 100%

**Comment by ghost** [comment]
Reread one of the emails you sent me and I had misread it originally. I will comment out the webwork parts for the 3D images.

**Comment by sean-fitzpatrick** [comment]
OK, I can look at the LaTeX commands. Are they failing in MBX?
Shouldn't be too hard to fix for someone who speaks LaTeX.
Thanks for the work on this. Looks good -- just the things I flagged to take care of.

File: src_pp/sec_polarcalc.ptx (hunk 1 above)
Comment (sean-fitzpatrick) [reply src_pp/sec_polarcalc.ptx:1914]: I put it back in.

File: src_pp/sec_series.ptx (hunk 3 above)
Comment (sean-fitzpatrick) [reply src_pp/sec_series.ptx:501]: Put width back in second case. For the first, put all three images together in a sidebyside.

File: src_pp/sec_taylor_poly.ptx (hunk 4 above)
Comment (sean-fitzpatrick) [reply src_pp/sec_taylor_poly.ptx]: I put them all back

**Review by sean-fitzpatrick — APPROVED** [review]
OK, I reverted the changes I didn't like.
Still not sure if triple integrals is going to merge correctly, but I made a backup copy.
I can put it back on the master branch if there are any problems.

**Pull request merged**